src/database/mockdata/articledata.js

```diff
@@ -0,0 +1,47 @@
+import slug from 'slug';
+import crypto from 'crypto';
+
+export const articleSample = {
+  publisheddate: new Date(),
```

Comment:
I do not think we need this. If the purpose for this is testing, can it be moved into the `helper/utils` file in the test folder? and also use faker to generate such data? is there any reason why this is on the `database folder`?

Reply:
Thanks for this review.
As at the time this was implemented, there was no `helper/utils` folder in the test folder. 
This is a new development. Thus, I will look at it and see how it will be implemented.